data/teams.yaml

```diff
@@ -91,6 +94,7 @@
     - Kexing Ying
     - Aaron Anderson
     - Kevin Buzzard
+    - Eric Rodriguez
```

Comment:
@ericrbg; do you want a middle name here?

Reply:
My second surname Boidi would be great :)

Reply:
Feel free to add a suggestion in github, then I'll commit and merge